canine/localization/base.py

```diff
@@ -879,7 +882,7 @@ def create_persistent_disk(self,
             # for scratch disk jobs. (we still need to mount the disk just in case
             # delocalization never ran).
             # use special exit code for this.
-            if finished:
+            if finished and self.scratch_disk_job_avoid:
```

Comment:
Add a clarifying comment here as well, consistent with the updated argument description above.

Reply:
Added in e541a45